packages/code-du-travail-frontend/src/outils/DureePreavisDemission/steps/__tests__/situation.test.js

```diff
@@ -1,114 +1,97 @@
 import {
+  getSituationsFor,
```

Comment:
you don't need to keep tests other than `recapSituation()` in this file.

Reply:
OK

Reply:
Done